README.rst

```diff
@@ -13,213 +13,10 @@ Features
 * Easy management of credentials.
 * Easy creation of Google Ads API service clients.
 
-Requirements
-------------
-* Python 3.7+
```

Comment:
You can keep this section. Quick reference info that is useful at a glance can be useful in the GitHub repo.

Reply:
Sounds good, in that case I think I'll add the one-line installation instructions too because that's very useful to have here.